ibis/tests/all/test_window.py

```diff
@@ -132,6 +132,12 @@
             lambda gb: gb.float_col.cummax(),
             id='max',
         ),
+        param(
+            lambda t: t.double_col.count(),
+            lambda gb: gb.double_col.cumcount(),
+            id='count',
+            marks=pytest.mark.xfail
```

Comment:
If this is passing on any backends, then you should use `pytest.mark.xfail_backends([Backend1, ..., BackendN])` to allow the passing backends to actually pass as opposed to `XPASS`ing.

Reply:
sounds good .. I will try it! thanks!